### PR TITLE
Fix use only first word of name when comparing to key

### PR DESCRIPTION
**Pull request by samuveth**

### CODE REVIEW

**Comment by samuveth** [comment]
[Fix Aragon plugin](https://trello.com/c/LyoEwrkZ/274-fix-aragon-plugin)

File: src/components/Modal/ProposalPlugins.vue [hunk 1]
```diff
@@ -28,7 +28,7 @@ function showButton(pluginObj) {
   );
   return pluginsWithParams
     .map(plugin => plugin.toLowerCase())
-    .includes(pluginObj.name.toLowerCase());
+    .includes(pluginObj.name.toLowerCase().split(' ')[0]);
```
Comment (bonustrack) [inline src/components/Modal/ProposalPlugins.vue:31]: I dont understand why you use name here? This look like an ugly fix, it should use the plugin key, we can't assume the name of a plugin will match the key. We don't need lowercase or anything.

Comment (samuveth) [reply src/components/Modal/ProposalPlugins.vue:31]: There isn't a plugin key as far as I can see.

Comment (bonustrack) [reply src/components/Modal/ProposalPlugins.vue:31]: These are the keys https://github.com/snapshot-labs/snapshot.js/blob/master/src/plugins/index.ts#L8-L12

Comment (samuveth) [reply src/components/Modal/ProposalPlugins.vue:31]: This is what the plugin object looks like `{author: "Gnosis", version: "1.0.0", name: "SafeSnap", website: "https://safe.gnosis.io"}`

Comment (samuveth) [reply src/components/Modal/ProposalPlugins.vue:31]: Oh I think I see what I could do.

**Review by bonustrack — APPROVED** [review]

**Pull request merged**